recipe/conda_build_config.yaml

```diff
@@ -1,2 +1,7 @@
 c_compiler:
   - gcc     # [linux]
+
+c_compiler:                    # [win]
+- vs2019                       # [win]
```

Comment:
We are compiling with clang, so this doesn't matter.

Reply:
Thanks for the explanation; I just don't want to open up the same inconsistency again, and also, I do not understand the interaction between clang-cl & MSVC, so erring on the side of caution.